src/appkit.cs

```diff
@@ -3731,6 +3731,7 @@ partial interface NSColor : NSCoding, NSCopying, NSSecureCoding, NSPasteboardRea
 		NSColor SelectedControl { get; }
 
 		[Static]
+		[Deprecated (PlatformName.MacOSX)]
```

Comment:
I used that due to `API_TO_BE_DEPRECATED` here:
```
@property (class, strong, readonly) NSColor *secondarySelectedControlColor NS_DEPRECATED_WITH_REPLACEMENT_MAC("unemphasizedSelectedContentBackgroundColor", 10_1, API_TO_BE_DEPRECATED);
```

Though, I could at least mention `UnemphasizedSelectedContentBackgroundColor` instead.

Reply:
hmm... what does `API_TO_BE_DEPRECATED` resolves to in Xcode 10.0 ?

Reply:
```#define __API_TO_BE_DEPRECATED 100000```

Reply:
ah! that's where that `10000` in the source comes from :)
I guess it's fine - at some point it will be updated and xtro will pick up the inconsistency right ?